pydantic/decorator.py

```diff
@@ -105,6 +106,7 @@ def __init__(self, function: 'AnyCallableT', config: 'ConfigType'):  # noqa C901
             elif p.kind == Parameter.POSITIONAL_OR_KEYWORD:
                 self.arg_mapping[i] = name
                 fields[name] = annotation, default
+                fields[V_DUPLICATE_KWARGS] = List[str], None
```

Comment:
This line is very strange, it's either some strange kung fu madness I don't understand, or a typo, do you mean:
```suggestion
                fields[V_DUPLICATE_KWARGS]: List[str] = None
```
?

If not requires a comment explaining it. If that is what you meant, I have no idea how it's passing both mypy and  tests 🙃 .

Reply:
This is inline with how `V_POSITIONAL_ONLY_NAME` works.

`fields` is a dictionary of annotations + defaults (as tuples), so I think everything here is as expected

Reply:
sorry, yes, my mistake, reading  the whole function makes more sense.